Readme.md

```diff
@@ -106,10 +106,19 @@ If you want to revert back to the original name, just clear the name and click r
 
 ### How can I help/contribute?
 If you are a coder, you are very welcome! You can fork this repo and then create a pull request.
+
+To check formats and errors of code before run the code:
+
+```shell
+make fmt
+
+make check
```

Comment:
It is fine to have the make file, but I would prefer that this would just call the cargo commands instead.

```suggestion
cargo fmt

cargo check
```

Reply:
Done.